### PR TITLE
add timer functionality

### LINKED ISSUE #12: Add game length timer
Have the engine track how long the game is not paused for, and have the visualizer display that time.

This is relevant if multiple competitors eat all of the pellets in the same number of lives, according to the rules.

### PR DESCRIPTION
**Pull request by thctamm**
resolves #12

### CODE REVIEW

File: 2017-2018/GameEngine/pacbot/gameState.py [hunk 1]
```diff
@@ -158,4 +168,6 @@ def restart(self):
         self.state_counter = 0
         self.update_ticks = 0
         self.lives = starting_lives
+        self.elapsed_time = 0
+        self.previous_start = time.time()
```
Comment (aolkin) [inline 2017-2018/GameEngine/pacbot/gameState.py:172]: Isn't this unnecessary since the game starts in a paused state?

Comment (thctamm) [reply 2017-2018/GameEngine/pacbot/gameState.py:172]: indeed

**Comment by aolkin** [comment]
Shame the robomodules renames didn't take properly...

**Comment by thctamm** [comment]
yup! I'll rewrite this. Won't be hard

**Pull request closed without merging**